benchmarking/runs/real_data/cosc_341/scenario1.py

```diff
@@ -0,0 +1,231 @@
+                                f"api/ai/external_algorithms/group_matcher_algorithm/group-matcher/inpData/{175}-generated.csv",
+                            )
+                        ),
+                        group_matcher_run_path=os.path.abspath(
+                            os.path.join(
+                                os.path.dirname(__file__),
+                                "../../../..",
+                                "api/ai/external_algorithms/group_matcher_algorithm/group-matcher/run.py",
+                            )
+                        ),
+                    ),
+                ],
+            },
+        ).run(num_runs=1)
+
+        artifacts.update(
+            SimulationSet(
+                settings=simulation_settings_1,
+                algorithm_set={
+                    AlgorithmType.PRIORITY: [
+                        PriorityAlgorithmConfig(
+                            MAX_TIME=1000000,
+                            MAX_KEEP=30,
+                            MAX_SPREAD=100,
+                            MAX_ITERATE=250,
+                        ),
+                    ],
+                    AlgorithmType.RANDOM: [
+                        RandomAlgorithmConfig(),
+                    ],
+                },
+            ).run(num_runs=num_trials)
+        )
+
+        artifacts.update(
+            SimulationSet(
+                settings=simulation_settings_2,
+                algorithm_set={
+                    AlgorithmType.PRIORITY: [
+                        PriorityAlgorithmConfig(
+                            MAX_TIME=1000000,
+                            MAX_KEEP=30,
+                            MAX_SPREAD=100,
+                            MAX_ITERATE=250,
+                            name="Switched Order Priority",
+                        ),
+                    ],
+                    AlgorithmType.RANDOM: [
+                        RandomAlgorithmConfig(),
+                    ],
+                },
+            ).run(num_runs=num_trials)
+        )
+
+        if compute_metrics:
+            graph_data: Dict[str, Dict[str, GraphData]] = {}
+            insight_set: Dict[str, InsightOutput] = Insight.get_output_set(
+                artifact=artifacts, metrics=list(metrics.values())
+            )
+
+            average_metrics: Dict[str, Dict[str, float]] = {}
+            for metric_name in metrics.keys():
+                average_metrics[metric_name] = Insight.average_metric(
+                    insight_set, metrics[metric_name].name
+                )
+
+            for metric_name, average_metric in average_metrics.items():
+                if metric_name not in graph_data:
+                    graph_data[metric_name] = {}
+                for algorithm_name, value in average_metric.items():
+                    if algorithm_name not in graph_data[metric_name]:
+                        graph_data[metric_name][algorithm_name] = GraphData(
+                            x_data=[175],
```

Comment:
at this point, 175 shows up so much that we should define it as a constant at the top of this function and give it a name (I'm assuming it's the class size)

Reply:
Yeah, it is. I can do that.